### PR TITLE
Add kokoro

### PR DESCRIPTION
**Pull request by busunkim96**
* Remove Appveyor
* Remove Coveralls
* Use nox

### CODE REVIEW

File: .kokoro/continuous.cfg [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Format: //devtools/kokoro/config/proto/build.proto
```
Comment (sduskis) [inline .kokoro/continuous.cfg]: Does this need an Apache header?

File: .kokoro/presubmit.cfg [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Format: //devtools/kokoro/config/proto/build.proto
```
Comment (sduskis) [inline .kokoro/presubmit.cfg]: header?

File: .kokoro/release.cfg [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# Format: //devtools/kokoro/config/proto/build.proto
```
Comment (sduskis) [inline .kokoro/release.cfg]: header?

**Comment by busunkim96** [comment]
@sduskis That sounds good, I'll separate out the PRs.

File: .kokoro/continuous.cfg (hunk 1 above)
Comment (busunkim96) [reply .kokoro/continuous.cfg]: I don't think so, Kokoro .cfg files in other repo's don't have them. ([google-cloud-python](https://github.com/googleapis/google-cloud-python/tree/master/.kokoro/continuous), [google-cloud-java](https://github.com/googleapis/google-cloud-java/tree/master/.kokoro))

**Pull request merged**